textual_fspicker/base_dialog.py

```diff
@@ -100,11 +112,14 @@ def compose(self) -> ComposeResult:
         """
         with Dialog() as dialog:
             dialog.border_title = self._title
+            currentDrive = str(os.path.splitdrive(MakePath.of(self._location).expanduser().absolute())[0]+'\\').upper()
+            yield Select.from_values(self.drives, value=currentDrive)
             yield DirectoryNavigation(self._location)
             with InputBar():
                 yield from self._input_bar()
-                yield Button(self._select_button, id="select")
-                yield Button("Cancel", id="cancel")
+                yield Button("Cancel", id="cancel", variant="error")
+                yield Button(self._select_button, id="select", variant="primary")
```

Comment:
This seems to swap the buttons around for no obvious reason; which would also be out of scope of this PR.

Reply:
Sorry for the switch of buttons was not intended